Content.Server/Power/EntitySystems/CableSystem.cs

```diff
@@ -17,7 +17,7 @@ public override void Initialize()
         base.Initialize();
 
         SubscribeLocalEvent<CableComponent, InteractUsingEvent>(OnInteractUsing);
-        SubscribeLocalEvent<CableComponent, CuttingFinishedEvent>(OnCableCut);
+        SubscribeLocalEvent<CuttingFinishedEvent>(OnCableCut);
```

Comment:
Is it possible to raise this directed?

Reply:
Not without changing toolsystem, which I've now done.